portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -9,6 +9,8 @@
   private final String CONTENT;
   private int numLikes;
   private final Date TIMESTAMP;
+  private final int ID;
+  private static int numComments = 0;
```

Comment:
All of these fields should be camelCase. Caps are used for static final variables:  https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names